app/views/course/assessment/submissions/_guided.html.slim

```diff
@@ -0,0 +1,3 @@
+div.btn-group
```

Comment:
Ok, and do you think we should display all question numbers here ?  Pagination skipped some numbers when it's too long, e.g `1 2 3 4...last`